scripts/bootstrap.sh

```diff
@@ -2,10 +2,32 @@
 
 set -e
 
-BASE_FOLDER=$1
+ENVIRONMENT=$1
+HOME_DIR=$2
 
-mkdir -p /tmp/.devopsloft
 apt-get update
 apt-get install -y mysql-client python3-pip
 
-pip3 install -r $BASE_FOLDER/requirements.txt
+# shellcheck source=/vagrant/.env
+source $HOME_DIR/.env
+if [[ -f $HOME_DIR/.env.local ]]; then
+# shellcheck source=/vagrant/.env.local
+  source $HOME_DIR/.env.local
+fi
+
+pip3 install --upgrade pip
+pip install -r $HOME_DIR/requirements.txt
```

Comment:
Why did you change pip3 to pip?

Reply:
As soon as pip3 upgrades, it sets pip to be pip3.

Reply:
+1

Reply:
removed line 18 because it caused issues of mixing system installer (apt-get) and pip